src/api.ts

```diff
@@ -26,30 +27,25 @@ export type BranchName = string;
 export type PortNumber = number;
 export type ImageStatus = 'NoImage' | 'Inactive' | PortNumber;
 
-// TODO: move out to configuration files
-export const BUILD_LOG_FILENAME = 'dserve-build-log.txt';
-export const REPO = 'Automattic/wp-calypso';
-export const TAG_PREFIX = 'dserve-wpcalypso';
-const CLONE_PREFIX = 'git@github.com:';
-
 export const ONE_SECOND = 1000;
 export const ONE_MINUTE = 60 * ONE_SECOND;
 export const FIVE_MINUTES = 5 * ONE_MINUTE;
 export const TEN_MINUTES = 10 * ONE_MINUTE;
 export const CONTAINER_EXPIRY_TIME = FIVE_MINUTES;
 
-export const getImageName = (hash: CommitHash) => `${TAG_PREFIX}:${hash}`;
+export const getImageName = (hash: CommitHash) => `${config.build.tagPrefix}:${hash}`;
 export const extractCommitFromImage = (imageName: string): CommitHash => imageName.split(':')[1];
 
 /**
  * Returns which images are stored locally.
  * Polls locker docker daemon for image list
  */
-export const { refreshLocalImages, getLocalImages } = (function() {
+export const { refreshLocalImages, getLocalImages } = (function () {
 	let localImages = {};
 	return {
 		refreshLocalImages: async () => {
 			localImages = _.keyBy(await docker.listImages(), 'RepoTags');
+			localImages = _.filter(localImages, (value, key) => key.startsWith(config.build.tagPrefix));
```

Comment:
nice bugfix

Reply:
I take it back. @dmsnell: this creates a bug. `localImages` needs to be an object keyed by RepoTags.  `_.filter` always returns an array.  you are looking for `_.omitBy`

Reply:
refactored in 58c89d9

I think we have a problem with tags - too many places treat RepoTags as a single string (or maybe it's just my own new code)

we lose type information when we use functions like `_.keyBy()` here because it doesn't tell us what `RepoTags` is. I'd favor constructs that preserve the type information, which could have fixed another bug here, which is the fact that we are building object keys out of arrays.

```js
// [ts] Type 'string[]' cannot be used as an index type.
const images = await docker.listImages()
images.forEach( image => {
	localImages[ image.RepoTags ] = {};
} )
```